### PR TITLE
add `Nonce` to `Block`, add `Random` to `Runtime`

### PR DESCRIPTION
**Pull request by Liaojinghui**

### CODE REVIEW

File: src/Neo.SmartContract.Framework/Services/Runtime.cs [hunk 1]
```diff
@@ -44,6 +44,12 @@ public static class Runtime
             get;
         }
 
+        public static extern ulong Random
```
Comment (shargon) [inline src/Neo.SmartContract.Framework/Services/Runtime.cs:47]: `GetRandom` better?

Comment (Liaojinghui) [reply src/Neo.SmartContract.Framework/Services/Runtime.cs:47]: Sure

File: src/Neo.SmartContract.Framework/Services/Runtime.cs [hunk 2]
```diff
@@ -44,6 +44,12 @@ public static class Runtime
             get;
         }
 
+        public static extern ulong GetRandom
```
Comment (shargon) [inline src/Neo.SmartContract.Framework/Services/Runtime.cs:47]: I mean like method, usually properties return the same value

**Review by nicolegys — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Comment by devhawk** [comment]
Can  we also add Runtime.Network? https://github.com/neo-project/neo/pull/2515

**Comment by shargon** [comment]
> Can we also add Runtime.Network? neo-project/neo#2515

It was added

**Review by chenzhitong — APPROVED** [review]

**Comment by superboyiii** [comment]
Merge? Works well on mine.

**Review by superboyiii — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**